jenkins/databricks/build.sh

```diff
@@ -47,8 +48,13 @@ CUDA_VERSION=`mvn help:evaluate -q -pl dist -Dexpression=cuda.version -DforceStd
 SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS=$BASE_SPARK_VERSION-databricks
 RAPIDS_BUILT_JAR=rapids-4-spark_$SCALA_VERSION-$SPARK_PLUGIN_JAR_VERSION.jar
 
+BUILD_ARG=""
+if [ -n "$STAGING_REPO" ]; then
+    BUILD_ARG="-s $WORKSPACE/jenkins/settings.xml -P"'sonatype-staging,!artifactory,!mirror-apache-to-urm,!deploy-to-urm'
```

Comment:
deactive all URM repos by -P'!artifactory,!mirror-apache-to-urm,!deploy-to-urm'
URM repo is unaccessiable in Databricks